.github/workflows/reusable-lint.yml

```
@@ -0,0 +1,19 @@
+name: Reusable Lint Workflow
+
+on: workflow_call
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2.4.0
+      - uses: actions/setup-node@v2.5.1
+        with:
+          node-version: "16.x"
+          registry-url: "https://registry.npmjs.org"
```

```suggestion
          cache: "npm"
```

Registry is unnecessary, cache should be used.